docs/architecture.md

```diff
@@ -1,37 +1,53 @@
 # Architecture
 
-**Note:** As with any young project, things change rapidly. Thus all the details in this architecture documentation may not be always up-to-date, but the high level concepts and patterns should still apply.
+**Note:** As k0s is a new and dynamic project, the product architecture may
```

Comment:
Remove the manual line changes (new line characters) from the middle of the paragraphs (this is general comment for all changes). Otherwise it becomes difficult to modify the text later on and you’d need to change all the line changes to different place. Line changes can be left to the text editor and browser, which will automatically scale the text according to the screen size. The whole docs has been written without manual line changes.